### PR TITLE
Add some style to the HTML tooltip

### PR DESCRIPTION
**Pull request by atabel**
I have extracted the arguments-hint rendering logic to 3 different classes, one for each strategy: status, panel and tooltip.

I have also added some html/style to the tooltip to distinguish argument names and types and highlight the current function argument:

style tooltip

### CODE REVIEW

File: tern.py [hunk 1]
```diff
@@ -453,71 +457,17 @@ def show_argument_hints(pfile, view):
   data = run_command(view, {"type": "type", "preferFunction": True}, call_start, silent=True)
   if data is not None:
     parsed = parse_function_type(data)
-    parsed['url'] = data.get('url', None)
-    parsed['doc'] = data.get('doc', None)
-    pfile.cached_arguments = (call_start, parsed)
-    render_argument_hints(pfile, view, parsed, argpos)
+    if parsed is not None:
```
Comment (atabel) [inline tern.py]: This fixes https://github.com/marijnh/tern_for_sublime/issues/72

Comment (marijnh) [inline tern.py]: Don't just throw fixes into non-related commits, through. It makes it hard to see what is really going on and, in this case, needlessly conflicts with another patch (075eaa79cfe5de5)

**Comment by marijnh** [comment]
Using ST build 3083, I'm not seeing these styles. Am I doing something wrong?

**Comment by atabel** [comment]
I'm using the last version of sublime 3088 in Linux (Debian). Everything seems to work fine, the styles look like you can see in the attached screenshot.

**Comment by marijnh** [comment]
I wasn't seeing the styles because I screwed up checking out the proper commit. I can see them now. Looks good. Merged as be8eb96

**Pull request closed without merging**